src/js/components/JobForm.js

```diff
@@ -43,21 +37,60 @@ class JobForm extends SchemaForm {
       return;
     }
 
-    this.validateForm();
-    // Provide updated model to get new multipleDefinition
-    let model = this.triggerTabFormSubmit();
-    this.multipleDefinition = this.getNewDefinition(model);
-    this.props.onChange(this.getDataTriple(model));
-    // Only update if schedule changed
-    if (scheduleEnabledChange) {
+    // Fetch the updated model from the <Form /> component
+    // nested deeper in the component tree.
+    this.model = this.triggerTabFormSubmit();
```

Comment:
I am sorry if I am late to the game here, I have been working pretty hard to not set `model` on this and just work with local variables. Is there a reason why you decided to set `model` on this? If not I would really like to see that not happen